SH4RE/SH4RE/CreateListing/CreateListingView.swift

```diff
@@ -226,6 +224,20 @@ struct CreateListingView: View {
             }
             else {
                 calAvail.append(availabilitySelection)
```

Comment:
maybe this should be moved to the end of the if, else if, .. , else {} just for clarity sake

Reply:
that's not supposed to be there, thanks for catching